.env.example

```
@@ -0,0 +1,20 @@
+LOG_CHANNEL=stack
+LOG_LEVEL=debug
+
+DB_CONNECTION=mysql
+DB_HOST=db
+DB_PORT=3306
+DB_DATABASE=centra_db
+DB_USERNAME=centra_user
+DB_PASSWORD=root
+
+REDIS_CLIENT=phpredis
+REDis_HOST=redis
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The REDis_HOST key should be in uppercase

(LowercaseKey)